src/bower_components/emby-apiclient/sync/serversync.js

```diff
@@ -12,11 +12,7 @@ define([], function() {
     }
 
     function uploadContent(connectionManager, server, options) {
-        return new Promise(function(resolve, reject) {
-            require(["contentuploader"], function(ContentUploader) {
-                (new ContentUploader).uploadImages(connectionManager, server).then(resolve, reject)
-            })
-        })
+        return new Promise().resolve();
```

Comment:
shouldn't we `reject` here instead?

Reply:
Sure, I was just filling it with something since it will hopefully be removed soon.